examples/core/instancing/app.js

```diff
@@ -133,16 +135,26 @@ class AppAnimationLoop extends AnimationLoop {
       }
     });
 
-    const timerElement = new TimerElement(this);
+    const statsWidget = new StatsWidget(this.stats, {
+      containerStyle: 'position: absolute;top: 20px;left: 20px;'
+    });
+    statsWidget.setFormatter('CPU Time', stat => `CPU Time: ${stat.getAverageTime().toFixed(2)}`);
+    statsWidget.setFormatter('GPU Time', stat => `GPU Time: ${stat.getAverageTime().toFixed(2)}`);
+    statsWidget.setFormatter('Frame Rate', stat => `Frame Rate: ${stat.getHz().toFixed(2)}fps`);
 
-    return {timerElement};
+    return {statsWidget};
   }
 
   onRender(animationProps) {
 
-    const {gl, framebuffer, useDevicePixels, _mousePosition, timerElement} = animationProps;
+    const {gl, framebuffer, useDevicePixels, _mousePosition, statsWidget} = animationProps;
 
-    timerElement.update();
+    if (widgetUpdateCount++ % 60 === 10) {
```

Comment:
You can use `animationProps.tick`

Reply:
Yes, AnimationLoop supports `tick` and `tock`. 
* One counts actual frames
* The other represents the number of frames that would have fired assuming 60 frames were fired per second (allows animations to remain unaffected by frame rate drops).